src/modules/remainingTime/index.ts

```diff
@@ -53,22 +53,22 @@ function RemainedTime(date: string | number | Date): RemainedTime & ToString & I
 		};
 	}
 
-	const Years = Math.floor(remainedTime / secondsInYear);
-	remainedTime %= secondsInYear;
+	const Years = Math.floor(remainingTime / secondsInYear);
+	remainingTime %= secondsInYear;
 
-	const Months = Math.floor(remainedTime / secondsInMonth);
-	remainedTime %= secondsInMonth;
+	const Months = Math.floor(remainingTime / secondsInMonth);
+	remainingTime %= secondsInMonth;
 
-	const Days = Math.floor(remainedTime / secondsInDay);
-	remainedTime %= secondsInDay;
+	const Days = Math.floor(remainingTime / secondsInDay);
+	remainingTime %= secondsInDay;
 
-	const Hours = Math.floor(remainedTime / secondsInHour);
-	remainedTime %= secondsInHour;
+	const Hours = Math.floor(remainingTime / secondsInHour);
+	remainingTime %= secondsInHour;
 
-	const Minutes = Math.floor(remainedTime / secondsInMinute);
-	remainedTime %= secondsInMinute;
+	const Minutes = Math.floor(remainingTime / secondsInMinute);
+	remainingTime %= secondsInMinute;
 
-	const Seconds = remainedTime;
+	const Seconds = remainingTime;
 
 	return {
 		Years,
```

Comment:
Please convert all return keys to `camelCase` here and everywhere you've written.